app/src/main/java/org/p2p/wallet/notification/FcmPushNotificationData.kt

```diff
@@ -1,6 +1,19 @@
 package org.p2p.wallet.notification
 
+import org.p2p.wallet.utils.emptyString
+import org.p2p.wallet.utils.find
+
 class FcmPushNotificationData(
     val title: String,
     val body: String,
+    val type: NotificationType = NotificationType.DEFAULT
 )
+
+enum class NotificationType(val type: String) {
```

Comment:
What is NotificationType for us? Do you mind calling me and explaining me this?

Reply:
it's type from server side
it could be `receive` for now
and `swap`... etc in future
it's synced with server notification type enum